src/main/java/org/ilgcc/jobs/TransmissionsRecurringJob.java

```diff
@@ -81,6 +86,13 @@ public void noProviderResponseJob() {
     }
 
     private boolean hasProviderResponse(Submission submission) {
```

Comment:
To clear up future confusion:
```suggestion
    private boolean hasProviderResponse(Submission familySubmission) {
```